experiments/process_sandbox/platform/syscall_context_freebsd.h

```diff
@@ -0,0 +1,153 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+#ifdef __FreeBSD__
+#  include <sys/syscall.h>
+#  include <ucontext.h>
+
+namespace sandbox::platform
+{
+#  ifdef __x86_64__
+  /**
+   * FreeBSD system call information.
+   *
+   * Note: This depends on FreeBSD D29185 landing.
```

Comment:
Perhaps leaving a URL instead of a bug number would help people unfamiliar with FreeBSD to reach it more easily.

How soon is that likely to land, and how soon is that likely to be part of an official release?